phoenix-shared/src/commonMain/sqldelight/paymentsdb/migrations/12.sqm

```diff
@@ -0,0 +1,32 @@
+-- Migration: v12 -> v13
+--
+-- Changes:
+-- * Added a new column [lightning_address] in table [payments_metadata]
```

Comment:
Needs comments for the new tables.

Reply:
Done in [9e7568c](https://github.com/ACINQ/phoenix/pull/640/commits/9e7568c9a5b25c7e0d4c39738bd832ea38e11370)